src/main/java/com/faforever/moderatorclient/ui/ViewHelper.java

```diff
@@ -1289,6 +1290,8 @@ public static void buildQuestionTable(TableView<VotingQuestionFX> tableView, Vot
         tableView.setEditable(true);
         HashMap<TableColumn<VotingQuestionFX, ?>, Function<VotingQuestionFX, ?>> extractors = new HashMap<>();
 
+        makeTableCopyable(tableView, true);
```

Comment:
Why are you calling it twice?

Reply:
I am not calling it twice! Please unfold the code completely. When looking at the code spinet it seems. but the two calls are in different methods...